.travis.yml

```diff
@@ -31,12 +31,8 @@ jobs:
     if: branch = master
     script: skip
     deploy:
-      - provider: deis
-        controller: https://deis.prod.revelry.net
-        app: slax
-        cli_version: v2.19.4
-        username: "$DEIS_USERNAME"
-        password: "$DEIS_PASSWORD"
+      - provider: script
+        script: echo 'Automatic deployment is configured directly in Heroku'
```

Comment:
I decided not to remove this entirely so there'd be a breadcrumb in the place where folks expect to find details about deployment.